docs/sources/installation/istio.md

```diff
@@ -0,0 +1,179 @@
+# Installation instructions for Istio
+
+The ingestor, querier, etc. might start, but if those changes are not made, you will see logs like
+
+```
+loki level=debug ts=2021-11-24T11:33:37.352544925Z caller=broadcast.go:48 msg="Invalidating forwarded broadcast" key=collectors/distributor version=123 oldVersion=122 content=[loki-distributor-59c4896444-t9t6g[] oldContent=[loki-distributor-59c4896444-t9t6g[]
+```
+
+This means that the pod is failing to join the ring.
+
+If you try to add `loki` to `Grafana` data sources, you will see logs like (`empty ring`)
+
+```
+loki level=warn ts=2021-11-24T08:02:42.08262122Z caller=logging.go:72 traceID=3fc821042d8ada1a orgID=fake msg="GET /loki/api/v1/labels?end=1637740962079859431&start=1637740361925000000 (500) 97.4µs Response: \"empty ring\\n\" ws: false; X-Scope-Orgid: fake; uber-trace-id: 3fc821042d8ada1a:1feed8872deea75c:1180f95a8235bb6c:0; "
+```
+
+When istio-injection is enabled on the namespace running Loki, there are few changes needed. One of the main changes is around the `Service` `appProtocols`. 
```

Comment:
"there are few changes needed."  It would be helpful to say here WHERE the changes are needed?  Istio configuration?  Pod configuration?  Loki configuration?   Reading the rest of this topic, it looks like we want to say something like:

"When you enable istio-injection on the namespace where Loki is running, you need to also modify the configuration for the Loki services.  Given that Istio will not allow a pod to resolve another mod using an IP address, you must also modify the `memberlist` service.